cli/tests/test_guardrail.py

```
@@ -2045,6 +2053,66 @@ def test_rejects_health_while_api_is_still_starting(self, connection_cls, mock_l
+            }
+        ).encode()
+        connection_cls.return_value.getresponse.side_effect = [retiring, replacement]
+        previous = _GatewayRuntimeGeneration(
+            b"retiring-pid-record",
+            "2026-08-11T04:00:00Z",
+            0.0,
+        )
+
+        self.assertTrue(
+            _wait_for_defense_gateway_api(
+                "/tmp/defenseclaw",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/defenseclaw"

(S108)